delivery_package_fee/tests/test_package_fee.py

```diff
@@ -94,10 +100,12 @@ def test_package_fee_simple(self):
         """All stock moves processed at once"""
         picking = self.sale.picking_ids
         self.assertEqual(picking.state, "assigned")
-        picking.move_line_ids[0].result_package_id = self.pack1
-        picking.move_line_ids[0].qty_done = 10.0
-        picking.move_line_ids[1].result_package_id = self.pack2
-        picking.move_line_ids[1].qty_done = 10.0
+        picking.move_line_ids[0].write(
+            {"result_package_id": self.pack1, "quantity": 10.0, "picked": True}
+        )
```

Comment:
why using quantity and picked to replace qty_done?

Reply:
https://github.com/odoo/odoo/pull/137864, we don't have `qty_done` anymore, we will use `quantity` and `picked`.

Reply:
ok, could you please mention it in PR's description